src/components/NpsSurvey.js

```diff
@@ -118,23 +118,24 @@ export const NpsSurvey = Utils.connectAtom(authStore, 'authState')(class NpsSurv
           boxShadow: Style.standardShadow
         }
       },
-      !expanded ?
-        'How are we doing?' :
-        [
-          div({ style: styles.questionLabel }, 'How likely are you to recommend Terra to others?'),
-          div({ style: { display: 'flex', justifyContent: 'space-around', marginBottom: '0.5rem' } }, scoreRadios),
-          div({ style: styles.questionLabel }, 'What was the reason for this score?'),
-          TextArea({ style: styles.questionInput, value: reasonComment, onChange: e => this.setState({ reasonComment: e.target.value }) }),
-          div({ style: styles.questionLabel }, 'What could we change?'),
-          TextArea({ style: styles.questionInput, value: changeComment, onChange: e => this.setState({ changeComment: e.target.value }) }),
-          div({ style: { display: 'flex', justifyContent: 'flex-end' } }, [
-            buttonSecondary({
-              style: { color: 'white' },
-              hover: { color: colors.gray[5] },
-              onClick: goAway(true)
-            }, 'Submit')
-          ])
-        ]
+      !expanded ? [
+        div({ style: styles.questionLabel }, 'How likely are you to recommend Terra to others?'),
+        div({ style: { display: 'flex', justifyContent: 'space-around', marginBottom: '0.5rem' } }, scoreRadios)
+      ] : [
+        div({ style: styles.questionLabel }, 'How likely are you to recommend Terra to others?'),
+        div({ style: { display: 'flex', justifyContent: 'space-around', marginBottom: '0.5rem' } }, scoreRadios),
+        span({ style: styles.questionLabel }, 'What was the reason for this score? '), span({ style: { ...styles.questionLabel, color: colors.gray[3] } }, '(Optional)'),
```

Comment:
I usually wrap inline elements (span) in a block level element (div) if the rest of the siblings are block level. I don't think it's critical, though. It all rendered correctly for me so it's probably fine in this case.

Reply:
I tried wrapping the `span`s in a `div` and it messed up the display, so I'm going to leave it as is.